datasets/UKNI01_northern_ireland/graph_values.yml

```diff
@@ -301,13 +301,13 @@ industry_final_demand_for_other_crude_oil_non_energetic:
 industry_useful_demand_for_chemical_refineries_crude_oil_non_energetic:
   demand: 0.0
 industry_final_demand_network_gas:
-  demand: 4709.837812
+  demand: 4238.85
 industry_chp_ultra_supercritical_coal:
   demand: 0.0
 industry_final_demand_crude_oil:
-  demand: 17026.79406
+  demand: 19592.73
```

Comment:
Correct? Most other demands go down

Reply:
Yes these are correct, thanks for checking. I will double-check with them tomorrow, but this is the data they have given me :)